### ISSUE TITLE
Add the ability to pass translations

### ISSUE THREAD

**Issue by yass0016** [Canadian-Geospatial-Platform/GeoView#67]
Add the ability to pass translation from outside within plugins

**Comment by jolevesq** [comment]
With the current viewer, we have a JSON object on the class who holds the translation;
```
Draw.prototype.translations = {
    'en-CA': {
        menu: 'Draw Toolbar',
        picker: 'Select colour',
        point: 'Draw point',
        line: 'Draw/Measure line',
        polygon: 'Draw/Measure polygon',
        edit: 'Edit existing drawing',
        measure: 'Show/Hide distances',
        extent: 'Erase selected graphics',
        write: 'Save graphics file',
        read: 'Upload graphics file'
    },
    'fr-CA': {
        menu: 'Barre de dessin',
        picker: 'Sélectionner la couleur',
        point: 'Dessiner point',
        line: 'Dessiner/Mesurer ligne',
        polygon: 'Dessiner/Mesurer polygon',
        edit: 'Éditer les dessins existant',
        measure: 'Afficher/Cacher les distances',
        extent: 'Effacer les graphiques sélectionnés',
        write: 'Sauvegarder le fichier de graphiques',
        read: 'Charger le fichier de graphiques'
    }
```

We need something similar and on load, this has to be merge with the translation obect used by i18n. then it will be avialable to use from the plugin html code

```
export const MAPNAV_DRAW_TOOLBAR_TEMPLATE = `
<span class="rv-spacer"></span>
<div class="rv-mapnav-content rv-mapnav-draw-content">
    <div class="rv-button-group hover rv-whiteframe-z2" ng-controller="DrawToolbarCtrl as ctrl">
        <md-button ng-repeat-start="control in ctrl.controls" name="{{ controls }}"
            aria-label="{{ control.label | translate }}"
            class="md-icon-button rv-button-32 rv-icon-16 rv-draw-button rv-draw-{{ control.name }}-button"
            ng-class="{ 'rv-control-active': control.selected() }"
            ng-click="control.action($event)">
            <md-tooltip md-direction="left">{{ control.tooltip | translate }}</md-tooltip>
            <md-icon>
                <svg xmlns="http://www.w3.org/2000/svg" fit="" height="100%" width="100%" preserveAspectRatio="xMidYMid meet" viewBox="0 0 24 24" focusable="false">
                <g id="drawicon{{control.name}}" ng-init="control.createIcon()"><path id="path{{control.name}}" d=""></path></g></svg>
            </md-icon>
        </md-button>
        <!-- this will insert divider after every element except the last one -->
        <md-divider ng-if="!$last" ng-repeat-end></md-divider>
        <input id="rvUploadGraphics" class="ng-hide" type="file" accept=".fgpv"></input>
    </div>
</div>
<span class="rv-spacer"></span>
`;
```

**Comment by jolevesq** [comment]
Closed by #50
